testapps/php56_extensions/composer.json

```diff
@@ -1,6 +1,33 @@
 {
     "require": {
         "php": "^5.6",
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "4.8.*",
```

Comment:
For example, if you put

```
"ext-memcached": "1.0.2"
```

The install script will enable the memcache extension, but I think the `composer install` will fail later. The build log says "The requested PHP extension ext-memcached 1.0.2 is missing from your system."

Does it make sense for us to rather say "We always use the latest version of extensions, please use "*" for your extension dependencies" or something?